src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -385,6 +389,27 @@ public void canDeclareItemLostIfLostPolicyChargeAmountMissing() {
     assertNoFeeAssignedForLoan(loan.getId());
   }
 
+  @Test
+  public void declaredLostEventIsPublished() {
+    final IndividualResource loanIndividualResource = checkOutFixture
+      .checkOutByBarcode(itemsFixture.basedUponNod(), usersFixture.jessica());
+
+    final DeclareItemLostRequestBuilder builder = new DeclareItemLostRequestBuilder()
+      .forLoanId(loanIndividualResource.getId())
+      .on(DateTime.now())
+      .withNoComment();
+    declareLostFixtures.declareItemLost(builder);
+
+    List<JsonObject> publishedEvents = Awaitility.await()
+      .atMost(1, TimeUnit.SECONDS)
+      .until(FakePubSub::getPublishedEvents, hasSize(2));
```

Comment:
Similarly to another comment, could a comment be added to explain why there should be two events or that the events are cleared before the item is declared lost.

Reply:
Added